solana/program_v2/packages/tests/src/pass-suite/change_pass_gatekeeper.test.ts

```diff
@@ -98,27 +109,78 @@ describe('Change pass gatekeeper', () => {
     expect(pass?.gatekeeper.toBase58()).to.equal(newDataAcct);
   });
 
-  it.skip('Cannot change to gatekeeper within a different network', async () => {
+  it('Cannot change to gatekeeper within a different network', async () => {
     // Assemble
-    const atlNetworkAuthority = Keypair.generate();
-    const [altStakingAccount] = await NetworkService.createStakingAddress(
-      atlNetworkAuthority.publicKey
+    const newAdminAuthority = Keypair.generate();
```

Comment:
You can reuse this function to do the set up :) `setUpAdminNetworkGatekeeper`

Reply:
Will attempt to do so!

Reply:
Done